### PR TITLE
Update maintainer status, add Will Beason

### PR DESCRIPTION
**Pull request by maxsmythe**
Re-up maintainer status for:

* sozercan
* ritazh
* smythe
* tsandall
* timothyhinrichs
* ashutosh-narkar

Swap willbeason in place of ctab

Leaving Oren to re-up when he's ready.

Signed-off-by: Max Smythe <smythe@google.com>

### CODE REVIEW

File: MAINTAINERS.md
Comment (tsandall) [inline MAINTAINERS.md:14]: nit: we were keeping the table in  alphabetical order (first name, last name). Could you move Will to the end? Thanks!

**Review by tsandall — COMMENTED** [review]
thanks @maxsmythe 🙏 

As a side note, once this is merged, I'd like to nominate @srenatus from Styra as a maintainer on the `opa` repo. @srenatus has been part of the community since 2018. Since then, he's shipped a number of valuable features (most recently, the `every` keyword) and has been an invaluable resource for the community when it comes to support.

**Review by sozercan — APPROVED** [review]
Thanks @maxsmythe and welcome @willbeason!

**Comment by maxsmythe** [comment]
Absolutely! Good call with the alphabetical thing, updated.

+1 to adding srenatus for OPA

**Review by willbeason — APPROVED** [review]

**Review by ritazh — APPROVED** [review]

**Pull request merged**
